source/use-cases/sensitive-data-encryption.txt

```diff
@@ -300,6 +289,17 @@ information:
 - The BSON Type of each field (only required for deterministically
   encrypted fields)
 
+.. admonition::JSON Schema Validation
+   :important:
+   
+   MongoDB drivers only use JSON Schema to configure automatic client-side
+   field level encryption. If you specify a document validation instance 
+   without an ``encrypt`` property, the client will either ignore it or throw
+   an error.
+
+   For information on how to use JSON Schema on the server to validate
```

Comment:
I think this may confuse users. Especially since they *can* specify encryption rules for server-side document validation. I would opt to remove this entirely, or re-frame as.

"Users *may* set server-side document validation using encryption keywords to enforce client-side field level encryption."

That said ,I've not yet completed the docs for that feature so it would be a dead link for now.

Reply:
Ok, I'll create a JIRA ticket to add the server-side document validation statement once we have a link. For now, i will remove mention of general schema validation.

Reply:
https://jira.mongodb.org/browse/DOCSP-7006